install/install.sh

```diff
@@ -0,0 +1,79 @@
+
+wget -O /etc/nginx/php_fastcgi \
+    https://gist.githubusercontent.com/cristiannsc/36c765c85daff7364e2c87639af399da/raw/php_fastcgi
+
+wget -O /etc/nginx/fastcgi_params \
+    https://gist.githubusercontent.com/cristiannsc/df341645f0cd2d8a9111f73b959e52f0/raw/fastcgi_params
+
+sudo ln -s /etc/nginx/sites-available/default /etc/nginx/sites-enabled/
+
+## Fixing some anoying Bug nginx + VirtualBox
+## http://wiki.nginx.org/Pitfalls
+## http://jeremyfelt.com/code/2013/01/08/clear-nginx-cache-in-vagrant/
+sed -i 's/sendfile on;/sendfile off;/g' /etc/nginx/nginx.conf
+
+sed -i 's/;cgi.fix_pathinfo=1/cgi.fix_pathinfo=1/g' /etc/php/7.2/fpm/php.ini
+
+sed -i 's/;extension=shmop/;extension=shmop\nextension=mcrypt.so\nextension=memcache.so/g' /etc/php/7.2/fpm/php.ini
+
+sudo service php7.2-fpm restart
+sudo service nginx restart
+
+echo "Configuring mySQL database\n"
+echo "Creating Meneame database\n"
+mysql -u root -p"rootpass" <<-CREATE_DATABASE
+  CREATE DATABASE meneame;
+  GRANT ALL ON meneame.* TO meneame@localhost IDENTIFIED BY 'meneame';
+  GRANT ALL ON meneame.* TO meneame@'%' IDENTIFIED BY 'meneame'
+CREATE_DATABASE
+
+echo "Creating tables\n"
+mysql -u meneame -p"meneame" meneame < meneame/sql/meneame.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2016-12-17-polls.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-02-24-strikes.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-04-27-links.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-08-04-preguntame.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-09-22-preguntame-patrocinado.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-09-26-sponsors.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2017-11-07-admin-users.sql
+mysql -u meneame -p"meneame" meneame < meneame/sql/2018-09-13-backups.sql
+
+echo "Creating initial data\n"
+mysql -u meneame -p"meneame" meneame <<-INITIAL_DATA
+  INSERT INTO subs (id,name,enabled,parent,server_name,base_url,name_long,visible,sub,meta,owner,nsfw,created_from,allow_main_link,color1,color2,private,show_admin,page_mode)
+    VALUES ( 1, 'mnm', 1, 0, '', '/', 'Principal', 1, 0, 0, 0, 0, 0, 1, NULL, NULL, 0, 0, NULL);
+  INSERT INTO sub_categories
+  	SELECT LAST_INSERT_ID(), category_id, 1, 1, 1, category_calculated_coef FROM categories;
+  INSERT INTO users (user_id,user_login,user_level,user_avatar,user_modification,user_date,user_validated_date,user_ip,user_pass,user_email,user_names,user_login_register,user_email_register,user_lang,user_comment_pref,user_karma,user_public_info,user_url,user_adcode,user_adchannel,user_phone)
+    VALUES (1,'Shadows236','god',0,'2018-10-10 21:41:28','2018-10-10 12:04:33','2018-10-10 21:42:28','127.0.0.1','sha256:JKVNPRNe5ZM2OvQ7Re42rUxYitMiMUMe:5d62fa6c578d2166b8b89732f0e8656ed54199df4fbafbc8ec32f8c0c858b2d6','admin@admin.cl','','Shadows236','admin@admin.cl',1,0,6.00,NULL,'',NULL,NULL,NULL);
```

Comment:
Why? Who is "Shadows236" and why does he need a superuser? Why are you inserting this data here on the first place?